mobile-browser-based-version/src/helpers/training/decentralised/training_informant.js

```diff
@@ -34,6 +32,15 @@ export class TrainingInformant {
     // training accuracy chart
     this.trainingAccuracyChart = null; // new TrainingChart("trainingAccuracy_".concat(taskID), "Training Accuracy")
     this.trainingAccuracy = null;
+
+    // is the model using Interoperability (default to false)
+    this.displayHeatmap = false;
+
+    // default values for the validation and training charts
+    this.currentValidationAccuracy = 0;
+    this.validationAccuracyDataSerie = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
```

Comment:
Even though the number of epochs is set to 10 it might change in the future, so let's a variable sized array for the initial chart:
let len = 10
this.validationAccuracyDataSerie = new Array(len).fill(0);